Singularity/Singularity/Game1.cs

```diff
@@ -80,53 +84,23 @@ protected override void Initialize()
         /// </summary>
         protected override void LoadContent()
         {
+            var viewportResolution = new Vector2(GraphicsDevice.Viewport.Width,
+                GraphicsDevice.Viewport.Height);
             // Create a new SpriteBatch, which can be used to draw textures.
             mSpriteBatch = new SpriteBatch(GraphicsDevice);
 
-            mMUnitSheet = Content.Load<Texture2D>("UnitSpriteSheet");
-            var mapBackground = Content.Load<Texture2D>("MockUpBackground");
-
-            // TODO: use this.Content to load your game content here
-            mPlatformBlankTexture = Content.Load<Texture2D>("PlatformBasic");
-            mPlatformDomeTexture = Content.Load<Texture2D>("Dome");
-            mPlatform = new PlatformBlank(new Vector2(300, 400), mPlatformBlankTexture);
-            mPlatform2 = new Junkyard(new Vector2(800, 600), mPlatformDomeTexture);
-            mPlatform3 = new EnergyFacility(new Vector2(600, 200), mPlatformDomeTexture);
+            mGameScreen = new GameScreen(mGraphics.GraphicsDevice.Viewport, mInputManager);
 
-            var fow = new FogOfWar(mapBackground);
-
-            mMap = new Map.Map(mapBackground, mGraphics.GraphicsDevice.Viewport, fow, mInputManager, true);
-
-            mMUnit1 = new MilitaryUnit(new Vector2(600, 600), mMUnitSheet, mMap.GetCamera(), mInputManager);
-            mMUnit2 = new MilitaryUnit(new Vector2(100, 600), mMUnitSheet, mMap.GetCamera(), mInputManager);
-
-            fow.AddRevealingObject(mMUnit1);
-            fow.AddRevealingObject(mMUnit2);
-
-            mMap.AddPlatform(mPlatform);
-            mMap.AddPlatform(mPlatform2);
-            mMap.AddPlatform(mPlatform3);
-
-            mGameScreen = new GameScreen(mMap);
-
-            // load roads
-            mRoad1 = new Road(mPlatform, mPlatform2, false);
-            var road2 = new Road(mPlatform, mPlatform3, false);
-            var road3 = new Road(mPlatform2, mPlatform3, false);
-
-            mGameScreen.AddObject(mMUnit1);
-            mGameScreen.AddObject(mMUnit2);
-            mGameScreen.AddObject(mPlatform);
-            mGameScreen.AddObject(mPlatform2);
-            mGameScreen.AddObject(mPlatform3);
-            mGameScreen.AddObject(mRoad1);
-            mGameScreen.AddObject(road2);
-            mGameScreen.AddObject(road3);
-            mGameScreen.AddObject(fow);
-            mGameScreen.AddObject(ResourceHelper.GetRandomlyDistributedResources(5));
+            mMainMenuManager = new MainMenuManagerScreen(viewportResolution, mScreenManager, true, this);
 
+            // Add the screens to the screen manager
+            // The idea is that the game screen is always at the bottom and stuff is added simply
```

Comment:
Why? Wouldnt it be better, if you only load the gamescreen (currently our object containing the gamestate) if you load the game?